quackfactory/cogs/session.py

```diff
@@ -0,0 +1,34 @@
+from discord.ext import commands
+import discord
+import logging
+
+intents = discord.Intents.default()
+intents.reactions = True
+intents.members = True
+client = discord.Client(intents=intents)
+
+class Session(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+@client.event
+async def on_voice_state_update(member, before, after):
+
+    guild = member.guild
+    role = "Test"
+    everyone = "@everyone" 
+    channel_id = 756574459556659323
+    message = f"Hey! {member} needs a rubber duck!"
+
+    if before.channel is None and after.channel.id == channel_id:

```

Comment:
Is there a reason you chose to only react to users who were not in other voice channels?